docs/index.html

```diff
@@ -52,6 +52,7 @@ <h2 class="title">A 1x Engineer...</h2>
       <section id="buttons" class="item">
         <a class="nes-btn" href="https://github.com/cutenode/1x.engineer">Open the Repository</a>
         <a class="nes-btn is-success" href="https://github.com/cutenode/1x.engineer/pulls">Send a Pull Request</a>
+        <a class="nes-btn is-primary" href="https://twitter.com/intent/tweet?url=https%3A%2F%2F1x.engineer&via=bitandbang&text=I%27m%20a%201x%20engineer%21&hashtags=1xEngineer" rel="noopener noreferrer">Tweet</a>
```

Comment:
I am super picky about click to tweets 😅

How would you feel about [this](https://twitter.com/intent/tweet?url=https%3A%2F%2F1x.engineer&text=✨%20I%27m%20a%201x%20engineer%20✨%20%0A) as the intent link?

```suggestion
        <a class="nes-btn is-primary" href="https://twitter.com/intent/tweet?url=https%3A%2F%2F1x.engineer&text=✨%20I%27m%20a%201x%20engineer%20✨%20%0A" rel="noopener noreferrer">Tweet</a>
```

Reply:
Absolutely fine by me! Will push a change 👍